app/CreateEvent/page.tsx

```
@@ -3,14 +3,24 @@ import React, { useState } from "react";
 import Header from "../components/Header";
 import BasicDetails from "../CreateEvent/components/BasicDetails/page";
 import Registration from "../CreateEvent/components/Registration/page";
-import Customization from "../CreateEvent/components/Customization/page"; 
-// import Footer from "../components/Footer";
+import Customization from "../CreateEvent/components/Customization/page";
 
 type Props = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)